tilt/Dockerfile

```diff
@@ -0,0 +1,35 @@
+ARG RUST_VERSION
```

Comment:
Aside from the configurable FEATURES, I essentially stole this from the soak/Dockerfile and soak/Dockerfile.build